app/src/main/java/com/termux/x11/input/TouchInputHandler.java

```diff
@@ -595,6 +595,10 @@ private boolean hasFlags(MotionEvent e, int flags) {
             return (e.getFlags() & flags) == flags;
         }
 
+        private boolean isScrollingEvent(MotionEvent e) {
+            return hasFlags(e, 0x14000000) || e.getClassification() == MotionEvent.CLASSIFICATION_TWO_FINGER_SWIPE;
```

Comment:
```suggestion
return hasFlags(e, 0x14000000) || (Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q && e.getClassification() == MotionEvent.CLASSIFICATION_TWO_FINGER_SWIPE);
```

Call requires API level 29 (current min is 26).